data/com/mojang/blaze3d/vertex/VertexFormat.mapping

```diff
@@ -8,7 +8,11 @@ CLASS com/mojang/blaze3d/vertex/VertexFormat
 			ARG 3 asGLMode
 			ARG 4 primitiveLength
 			ARG 5 primitiveStride
+		METHOD indexCount (I)I
+			ARG 1 vertexes
```

Comment:
For indexCount the word "vertexes" is correct for the input, because the input parameter is the number of vertexes being passed in to get a proper calculated amount. Changing to Vertices though.